yarn.lock

```diff
@@ -2663,7 +2663,7 @@ punycode@^1.3.2:
   resolved "https://registry.yarnpkg.com/punycode/-/punycode-1.4.1.tgz#c0d5a63b2718800ad8e1eb0fa5269c84dd41845e"
   integrity sha1-wNWmOycYgArY4esPpSachN1BhF4=
 
-qs@6.11.0, qs@^6.4.0, qs@^6.7.3, qs@^6.9.7:
```

Comment:
what's that ?

Reply:
Absolutely no idea.

Looking at git blame, it appears I'm responsible for this change. Let me revert it.

Reply:
Every time I modify this lock file in my local dev env, some format checks or other causes it to revert.

For safeties sake, I've simply used the git web editor to revert this change directly.